web/src/styles/global-style.ts

```diff
@@ -8,6 +8,12 @@ export const GlobalStyle = createGlobalStyle`
     --toastify-color-error: ${({ theme }) => theme.error};
   }
 
+  .react-loading-skeleton {
```

Comment:
I added this before @kemuru 's PR, so I think this is duplicated. Change the colors here with the ones at the bottom of the file, and leave only this definition.